core/client/fs/src/main/java/alluxio/client/file/BaseFileSystem.java

```diff
@@ -191,11 +190,7 @@ public FileOutStream createFile(AlluxioURI path, CreateFilePOptions options)
     FileSystemMasterClient masterClient = mFsContext.acquireMasterClient();
     URIStatus status;
     try {
-      masterClient.createFile(path, options);
-      // Do not sync before this getStatus, since the UFS file is expected to not exist.
-      GetStatusPOptions gsOptions = FileSystemOptions.getStatusDefaults(mFsContext.getConf())
-          .toBuilder().setLoadMetadataType(LoadMetadataPType.NEVER).build();
-      status = masterClient.getStatus(path, gsOptions);
+      status = masterClient.createFile(path, options);
```

Comment:
do we need to set the load metadata type?

Reply:
I don't think so, previously we set the load metadata type in `getStatus` because we implicitly knew it should be a new file which would not be shadowed in UFS. The create step should still check against the UFS based on the user's sync policy